posthog/models/insight.py

```diff
@@ -35,6 +35,7 @@ class Insight(models.Model):
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
     filters: models.JSONField = models.JSONField(default=dict)
     filters_hash: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    query: models.JSONField = models.JSONField(null=True)
```

Comment:
```suggestion
    query: models.JSONField = models.JSONField(null=True, blank=True)
```
So that admins can edit an insight without adding a query. Seems to be that layouts is missing the same, so maybe we're doing this on purpose? (Django newbee here)

Reply:
I'd never checked what the difference was 🤣 #til